enterprise/javaee.beanvalidation/build.xml

```diff
@@ -23,8 +23,6 @@
     <description>Builds, tests, and runs the project org.netbeans.modules.javaee.beanvalidation</description>
     <import file="../../nbbuild/templates/projectized.xml"/>
 
-    <property name="resources.dir" value="./src/org/netbeans/modules/javaee/beanvalidation/resources/"/>
     <target name="build-init" depends="projectized.build-init">
-        <unzip src="external/javaee.beanvalidation-resources.zip" dest="${resources.dir}"/>
     </target>
```

Comment:
Sorry - I missed this in the first round - lines 26 + 27 can be removed to. The qualified call to the imported target is only needed if the target is overriden. With that change this should be good. The build failure is caused by hg.netbeans.org being not reachable.